### ISSUE TITLE
Project abandoned -> welcome to use Simdkalman

### ISSUE THREAD

**Issue by oseiskar** [pykalman/pykalman#86]
As this project has clearly been mothballed since 2015, I hope nobody takes offence if I use this forum to welcome everyone to use and contribute to my alternative Python Kalman filter implementation, _Simdkalman_ https://github.com/oseiskar/simdkalman (https://pypi.org/project/simdkalman/).

I does not currently support EKF & UKF but can be up to a 100 times faster than Pykalman.

**Comment by duckworthd** [comment]
Very cool work, Otto!

As the original developer of `pykalman`, I admit that I have not had the time necessary to properly support this project as the community migrates to Py3. The original work is meant to be a proof of concept for others to build against. While I'm not familiar enough with `simdkalman` to wholeheartedly say others should use it instead of `pykalman`, I agree that the community deserves a better maintainer than myself.

Leaving issue open for FYI.

**Comment by oseiskar** [comment]
Thanks Daniel!

I wasn't really expecting a quick answer and was clearly too quick to judge the project as fully abandoned.

I also don't think that people should stop using this project as its functional and feature complete (as well as significantly more used battle-tested than my project). However, pull requests and issues might be better submitted somewhere else. To be fair, also **[filterpy](https://github.com/rlabbe/filterpy)** is definitely worth checking out.

`pykalman` is a great package and fully understand and agree that starting an open source project is not a promise to maintain it indefinitely for free.

Best regards,
Otto

**Comment by sanjayankur31** [comment]
Thanks for that @oseiskar 

@duckworthd : maybe it's worth posting a new ticket asking if anyone would like to take up maintainership? There seem to be pull requests and quite a few issues, so people are still using pykalman. Perhaps a few people would be willing to take it up?

**Comment by kalbasit** [comment]
I agree with @sanjayankur31 on this. @duckworthd if you're no longer going to be maintaining this project, perhaps you could add a note to the README noting the status of the project and possible alternatives to it. You could also open it up for volunteers to take on the maintenance. Thank you!

**Comment by markatango** [comment]
My two cents:
I'm (happily) using `pykalman` in an algorithm for detecting anomalies in a certain class of industrial process data.  In this case, the observation time series comprises several elements that are not independent, hence I think `pykalman` may be better suited than `simdkalman`
For my purpose,  I will be extending my fork of `pykalman` to include an adaptive EKF method to estimate process noise at each step.  Thank you, @duckworthd, for this outstanding project!
